src/tensorflow.cc

```diff
@@ -1731,6 +1730,7 @@ ModelState::AutoCompleteConfig()
 
     auto ach = AutoCompleteHelper(this, tritontf_model);
     RETURN_IF_ERROR(ach.Fix());
+    RETURN_IF_ERROR(SetModelConfig());
```

Comment:
For scalar inputs do we need to avoid calling `SetModelConfig`?

Reply:
I think we will still need to call `SetModelConfig` for models with scalar I/O. Only one of the input can be scalar and need to be provided explicitly in config. Also, we can not differentiate between unknown rank and scalar at this point. Explicit model config is something we rely on. If there are no clear factors, TF auto-complete logic is designed to handle such cases and throw an error.

Reply:
https://github.com/triton-inference-server/tensorflow_backend/blob/main/src/tensorflow.cc#L1627